pkg/webhook/propagationpolicy/mutating.go

```diff
@@ -36,8 +38,13 @@ func (a *MutatingAdmission) Handle(ctx context.Context, req admission.Request) a
 	// Set default namespace for all resource selector if not set.
 	for i := range policy.Spec.ResourceSelectors {
 		if len(policy.Spec.ResourceSelectors[i].Namespace) == 0 {
-			klog.Infof("Setting resource selector default namespace for policy: %s/%s", policy.Namespace, policy.Name)
-			policy.Spec.ResourceSelectors[i].Namespace = policy.Namespace
```

Comment:
Wow, I just realized the policy doesn't have a default namespace yet by this time.

Reply:
Do you know when the `kube-apiserver` would set the default namespace for CR resources?